common/network-shuffle/src/main/java/org/apache/spark/network/shuffle/protocol/FetchShuffleBlocks.java

```diff
@@ -86,6 +91,7 @@ public int hashCode() {
     result = 31 * result + shuffleId;
     result = 31 * result + Arrays.hashCode(mapIds);
     result = 31 * result + Arrays.deepHashCode(reduceIds);
+    result = 31 * result + shuffleGenerationId;
```

Comment:
move it right after `result = 31 * result + shuffleId;`?

Reply:
Thanks, done in 4c4af33.